test/__snapshots__/multi-client.test.js.snap

```diff
@@ -0,0 +1,276 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Client gets bundled css file from build server 1`] = `
+"body {
+    margin: 0;
+    padding: 0;
+}
+
+.b-title {
+    color: red;
+}body {
```

Comment:
I think you mentioned this at standup, but could the different css files be concatenated with a couple of linebreaks?

Reply:
yup, needs to be fixed in `asset-pipe-css-writer`. Will fix it like its hawt.